### PR TITLE
Resiliency changes

### PR DESCRIPTION
**Pull request by vishdivg**
#### What this PR does / why we need it:
It includes changes for PR 1052 as well some other changes for chaos engineering module in observability section
#### Which issue(s) this PR fixes:

Fixes #

#### Quality checks

- [ ] My content adheres to the style guidelines
- [ ] I ran `make test module="<module>"` it was successful (see https://github.com/aws-samples/eks-workshop-v2/blob/main/docs/automated_tests.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: website/docs/observability/high-availability/03-node-failure-no-fis.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+:::
+
+## Conclusion
+
+This node failure simulation demonstrates the robustness and self-healing capabilities of your Kubernetes cluster. Key observations and lessons from this experiment include:
+
+1. Kubernetes' ability to quickly detect node failures and respond accordingly.
+2. The automatic rescheduling of pods from the failed node to healthy nodes, ensuring continuity of service.
+3. The cluster's self-healing process, bringing the failed node back online after a short period.
```
[the remark this answers is not in the document]
Comment (vishdivg) [reply website/docs/observability/high-availability/03-node-failure-no-fis.md:124]: changed to below -:

File: website/docs/observability/high-availability/02-pod-failure.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+Once ready, you can access the retail store through this URL to confirm that it's still functioning correctly despite the simulated pod failure.
+
+## Conclusion
+
+This pod failure simulation demonstrates the resilience of your Kubernetes-based application. By intentionally causing a pod to fail, you can observe:
+
+1. The system's ability to detect failures quickly
+2. Kubernetes' automatic rescheduling and recovery of failed pods
```
[the remark this answers is not in the document]
Comment (vishdivg) [reply website/docs/observability/high-availability/02-pod-failure.md:98]: Kubernetes' automatic rescheduling and recovery of deployments or statefulsets failed pods.

File: website/docs/observability/high-availability/03-node-failure-no-fis.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+$ kubectl rollout status -n catalog deployment/catalog --timeout 180s
+$ kubectl rollout status -n orders deployment/orders-mysql --timeout 180s
+$ kubectl rollout status -n orders deployment/orders --timeout 180s
+$ timeout 10s $SCRIPT_DIR/get-pods-by-az.sh | head -n 30
+```
+
+These commands perform the following actions:
+
+1. Delete the existing ui pods.
+2. Wait for ui pods to get provision automatically.
```
[the remark this answers is not in the document]
Comment (vishdivg) [reply website/docs/observability/high-availability/03-node-failure-no-fis.md:87]: corrected-:
 Wait for ui pods to be provisioned automatically.

File: website/docs/observability/high-availability/01-scale.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+NAME: chaos-mesh
+LAST DEPLOYED: Tue Aug 20 04:44:31 2024
+NAMESPACE: chaos-mesh
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+
+```
+
+## Scaling and Pod Anti-Affinity
```
[the remark this answers is not in the document]
Comment (vishdivg) [reply website/docs/observability/high-availability/01-scale.md:34]: Changed-:
# Scaling and Topology Spread Constraints

File: website/docs/observability/high-availability/01-scale.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+We use a Kustomize patch to modify the UI deployment, scaling it to 5 replicas and adding pod anti-affinity rules. This ensures UI pods are distributed across different nodes, reducing the impact of node failures.
```
[the remark this answers is not in the document]
Comment (vishdivg) [reply website/docs/observability/high-availability/01-scale.md:36]: Thanks for pointing it out. Tested with topology spread constraint only and pods are getting distributed evenly on all three AZs.
changed scale_and_affinity_patch.yaml to have only topology spread constraints.

**Review by svennam92 — APPROVED** [review]
LGTM! Thank you @cyturney and @vishdivg for this PR, and @bbgu1 for reviewing.

**Review by svennam92 — APPROVED** [review]

**Pull request merged**